docs/web/interceptors.mdx

```diff
@@ -152,3 +152,40 @@ import { elizaClient } from "./eliza-client.js";
 
 const res = elizaClient.say({ sentence: "Hey!" }, { contextValues: createContextValues().set(kLogBody, true) });
 ```
+
+## Customizing Fetch
```

Comment:
Can we call this "Setting fetch() options" ?

We want to explain that `request.init` can be used to set optional parameters - with the exception of the ones reserved by connect-es. This is separate from bringing your own implementation of fetch(), which we call "custom fetch" at other places. So I think the distinction is more clear when we avoid "custom" here.

`keepalive` is fair to mention, we just don't want anyone to think it's about HTTP keep-alive. That's easy to do by mentioning that it's the replacement for Navigator.sendBeacon. I think that a more practical example is `redirect`, which we set to `error`, and which users may want to change.

I don't think a full runnable example is necessary here. It duplicates 90% of the example above.